packages/eslint-plugin-next/lib/rules/no-img-element.ts

```diff
@@ -41,3 +36,6 @@ module.exports = {
     }
   },
 }
+
+export default rule
+module.exports = rule
```

Comment:
This is ugly, but ESLint [only](https://eslint.org/docs/latest/developer-guide/working-with-rules) [accepts](https://github.com/eslint/eslint/issues/13232) `module.exports`, our `pnpm test-types` expects `export default`, and SWC cannot seem to output `module.exports`, but `exports.default`